easybuild/scripts/bootstrap_eb.py

```diff
@@ -448,16 +511,18 @@ def main():
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
 source_urls = [
-               'http://pypi.python.org/packages/source/e/easybuild-framework/',
-               'http://pypi.python.org/packages/source/e/easybuild-easyblocks/',
-               'http://pypi.python.org/packages/source/e/easybuild-easyconfigs/',
-              ]
+    'https://pypi.python.org/packages/source/v/vsc-base/',
```

Comment:
that'll make the names of the source tarballs trickier... we need separate tarballs for each of the packages, and naming them all `develop.tar.gz` is not going to work